spec/features/user_following_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+
+feature "User following" do
+  scenario "User follows and unfollow someone" do
+    alice = Fabricate(:user)
+    category = Fabricate(:category)
+    video = Fabricate(:video, category: category)
+    Fabricate(:review, video: video, user: alice)
+    sign_in
+    click_on_video_on_home_page(video)
+
+    click_link alice.full_name
+    click_link "Follow"
+    expect(page).to have_content(alice.full_name)
+
+    unfollow(alice)
+    expect(page).not_to have_content(alice.full_name)
+  end
+
+  def unfollow(user)
+    find("a[data-method='delete']").click
```

Comment:
Make sure you are more specific here - I would recommend using the `title` attribute on the Unfollow anchor element.